examples/multi_python_versions/requirements/requirements_lock_3_11.txt

```diff
@@ -1,10 +1,10 @@
 #
-# This file is autogenerated by pip-compile with Python 3.11
+# This file is autogenerated by pip-compile with Python 3.9
```

Comment:
I think this file should be generated with Python 3.11.

Reply:
Done